modules/sdk-coin-avaxp/package.json

```diff
@@ -51,6 +51,7 @@
     "@bitgo/sdk-core": "^26.3.0",
     "@bitgo/statics": "^48.1.0",
     "@bitgo/utxo-lib": "^9.34.0",
+    "bitgo-aaron-avalanchejs": "^4.0.5",
```

Comment:
This needs to be 
```suggestion
    "@bitgo/avalanchejs": "1.0.0-alpha.3",
```

Reply:
Thanks @andrew-scott-fischer. Will use this in PR - https://github.com/BitGo/BitGoJS/pull/4366 too.